client_admin/pulp/client/admin/tasks.py

```diff
@@ -202,7 +202,7 @@ def parse_state(task):
                 finish_time = _('N/A')
                 result = _('N/A')
             elif task.was_cancelled():
-                state = _('Cancelled')
+                state = _('Canceled')
```

Comment:
Ahhhhh, perhaps this is why I thought the API was "cancelled" - the CLI was printing "Cancelled" as the task status and I assumed it was that stupid CLI thing where it capitalizes words from the API when printing them. It turns out that we had code that detected "canceled" and then printed "cancelled"! lol.